test/unit-test/metadata/fam_metadata_test.cpp

```diff
@@ -76,6 +74,8 @@ int main(int argc, char *argv[]) {
         return TEST_SKIP_STATUS;
     }
 
+    Fam_Metadata_Service *manager = new Fam_Metadata_Service_Direct(true);
+
```

Comment:
metadata service is independent of openfam client-side code.
its not clear why we need this change.